src/lib/holocene/drawer.svelte

```diff
@@ -29,6 +30,7 @@
     {id}
     role="region"
     on:click-outside={onClick}
+    use:focusTrap={true}
```

Comment:
There is an edge case with the `Configure Workflows List`. When you hit `Enter` on the last item in the list to add it to the headings the drawer loses focus. Not sure on an elegant solution here, since usually that would be expected behavior (e.g. a close button). 

We could consider adding some optional param e.g. `exitOnEnter` and adding something like this to `focusTrap` ⬇️ 
```js
if (!exitOnEnter && event.key === 'Enter' && document.activeElement === lastFocusable) {
    firstFocusable.focus();
    event.preventDefault();
}
```